actions/class.Creator.php

```diff
@@ -215,4 +228,46 @@ protected function getRuntimeConfig()
             ->getExtensionById('taoQtiTest');
         return $extension->getConfig('testRunner');
     }
+
+    private function getRemoteListService(): ListService
+    {
+        return $this->getServiceManager()->getContainer()->get(ListService::class);
+    }
+
+    private function isScaleEnabled(): bool
+    {
+        return $this->getRemoteScaleListService()->isRemoteListEnabled();
```

Comment:
It will not be hidden behind FF? There will be no situation where we would want to set the remote list and still not show scales? Just curious.

Reply:
This is a feature flag. If we will not set it up remote list should not be created.